composer.json

```diff
@@ -21,18 +21,18 @@
         "aeon-php/rate-limiter": ">=0.7.0",
         "aeon-php/retry": ">=0.6.0",
         "aeon-php/sleep": "^1.0",
-        "symfony/config": "^4.4.12|^5.0",
-        "symfony/dependency-injection": "^4.4.12|^5.0",
-        "symfony/form": "^4.4|^5.0",
-        "symfony/http-kernel": "^4.4|^5.0",
-        "symfony/validator": "^4.4|^5.0"
```

Comment:
Dropping 4.4 is a bit problematic, I still have one project that is using this bundle and Symfony 4.4 

One thing we could do to allow whole range from 4.4 up to 6.0 would be to detect if `service` function is available through [function_exists](https://www.php.net/manual/en/function.function-exists.php) and use it, otherwise fallback to `ref`. (it could be all packed into `service` function in Aeon Bundle namespace `Aeon\Symfony\AeonBundle\DependencyInjection\Loader\Configurator`)

Reply:
Sure thing, I'll have a look into it today.

Reply:
Now passing tests on SF 4.4 with `--prefer-lowest`. Let me know if you'd like changes to any of the approaches I've taken.

Reply:
It's perfect, could you just take care of static analysis? 
🙌

Reply:
If it's complaining about anti BC break layer you added, feel free to suppress it sine that layer will get removed anyway

Reply:
I may be wrong, and it may be Sunday, but the SA reports looked like false-positives. So the latest try is suppressing the specific violations.

Reply:
btw. you can test whole CI build locally by running “composer build”